testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -1788,6 +1789,150 @@ QUnit.module('selection', moduleSetup, () => {
     });
 });
 
+QUnit.module('selectByClick option', {
+    beforeEach: function() {
+        this.onSelectionChangedSpy = sinon.spy();
+        this.items = [1, 2];
+        this.createList = (selectionMode, selectByClick) => {
+            return $('#list').dxList({
+                items: this.items,
+                showSelectionControls: true,
+                selectByClick,
+                selectionMode,
+                onSelectionChanged: this.onSelectionChangedSpy,
+            });
+        };
+    } }, ()=> {
+
+    QUnit.test('onSelectionChanged event should call on item click if selectionMode=single, selectByClick=true', function(assert) {
```

Comment:
let's create 2 submodules to separate cases more explicitly and simplify test names?
```javascript
QUnit.module('in single selection mode')
QUnit.module('in multiple selection mode');
```

Reply:
Fixed